manifests/compiler.pp

```diff
@@ -1,6 +1,10 @@
+# == Class homebrew::compiler
+# 
+# Installs Command Line Tools from user provided package if OS version > 10.7
+# 
 class homebrew::compiler {
 
-  if $::has_compiler == "true" {
+  if $::has_compiler == true {
```

Comment:
Agreed we want to avoid quoted boolean strings, but just want to make sure we definitely test this.